<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActionsWidget</class>
 <widget class="QWidget" name="ActionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>825</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>УЕТ действия:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="lblActionUET">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="text">
           <string>0.0</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>УЕТ всего:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="lblEventUET">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="text">
           <string>0.0</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="attrsAP">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="7">
         <widget class="CPersonComboBoxEx" name="cmbAPSetPerson">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>Врач</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="lblAPEndDate">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Выполнено</string>
          </property>
         </widget>
        </item>
        <item row="3" column="7">
         <widget class="CPersonComboBoxEx" name="cmbAPPerson">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>Врач</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="lblAPBegDate">
          <property name="text">
           <string>Начато</string>
          </property>
         </widget>
        </item>
        <item row="3" column="6">
         <widget class="QLabel" name="lblAPPerson">
          <property name="text">
           <string>Исполнитель</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="CDateEdit" name="edtAPEndDate">
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTimeEdit" name="edtAPEndTime">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="displayFormat">
             <string>HH:mm</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="6">
         <widget class="QLabel" name="lblAPSetPerson">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Назначил</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="CDateEdit" name="edtAPBegDate">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="whatsThis">
             <string>дата окончания осмотра</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTimeEdit" name="edtAPBegTime">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="displayFormat">
             <string>HH:mm</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="8">
         <widget class="QLabel" name="lblAPStatus">
          <property name="text">
           <string>Состояние</string>
          </property>
         </widget>
        </item>
        <item row="0" column="9">
         <widget class="QComboBox" name="cmbAPStatus">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="3" column="8">
         <widget class="QLabel" name="lblMKB">
          <property name="text">
           <string>МКБ</string>
          </property>
         </widget>
        </item>
        <item row="3" column="9">
         <widget class="CICDCodeEditEx" name="cmbMKB"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>912</width>
        <height>715</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QSplitter" name="splitterAP">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="childrenCollapsible">
          <bool>false</bool>
         </property>
         <widget class="CActionsTreeView" name="tblAPActions">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="animated">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QFrame" name="frameAP">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <property name="margin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>4</number>
           </property>
           <item row="18" column="0" colspan="2">
            <widget class="CExActionPropertiesTableView" name="tblAPProps">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>250</height>
              </size>
             </property>
             <property name="verticalScrollMode">
              <enum>QAbstractItemView::ScrollPerPixel</enum>
             </property>
             <property name="horizontalScrollMode">
              <enum>QAbstractItemView::ScrollPerPixel</enum>
             </property>
            </widget>
           </item>
           <item row="19" column="0" colspan="2">
            <widget class="QFrame" name="frmAPButtonsBar">
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <property name="lineWidth">
              <number>0</number>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <property name="spacing">
               <number>4</number>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
              <item>
               <widget class="CPrintButton" name="btnAPPrint">
                <property name="text">
                 <string>Печать</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="CActionTemplateChooseButton" name="btnAPLoadTemplate">
                <property name="text">
                 <string>Загрузить шаблон</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnAPSaveAsTemplate">
                <property name="text">
                 <string>Сохранить шаблон</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnAPLoadPrevAction">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Копировать из предыдущего</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>145</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit.h</header>
  </customwidget>
  <customwidget>
   <class>CPersonComboBoxEx</class>
   <extends>QComboBox</extends>
   <header>Orgs/PersonComboBoxEx.h</header>
  </customwidget>
  <customwidget>
   <class>CICDCodeEditEx</class>
   <extends>QComboBox</extends>
   <header>library/ICDCodeEdit.h</header>
  </customwidget>
  <customwidget>
   <class>CActionTemplateChooseButton</class>
   <extends>QPushButton</extends>
   <header>Events/ActionTemplateChoose.h</header>
  </customwidget>
  <customwidget>
   <class>CPrintButton</class>
   <extends>QPushButton</extends>
   <header>library/PrintTemplates.h</header>
  </customwidget>
  <customwidget>
   <class>CExActionPropertiesTableView</class>
   <extends>QTableView</extends>
   <header>Events/ActionPropertiesTable.h</header>
  </customwidget>
  <customwidget>
   <class>CActionsTreeView</class>
   <extends>QTreeView</extends>
   <header>Events/ActionsTreeModel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
